scout/server/blueprints/panels/views.py

```diff
@@ -311,3 +351,42 @@ def gene_edit(panel_id, hgnc_id):
                 if panel_value is not None:
                     form_field.process_data(panel_value)
     return dict(panel=panel_obj, form=form, gene=hgnc_gene, panel_gene=panel_gene)
+
+
+def _compile_panel_versions(panels_found):
```

Comment:
You don't need these 2 functions. Just optimize the query instead

Reply:
Done!